.github/workflows/finishReleaseCycle.yml

```diff
@@ -33,15 +33,15 @@ jobs:
             Reopening!
 
       - name: Check for any deploy blockers
-        if: ${{ steps.validateActor.outputs.isTeamMember }}
+        if: ${{ steps.validateActor.outputs.isTeamMember == 'true' }}
         id: checkDeployBlockers
         uses: Expensify/App/.github/actions/checkDeployBlockers@main
         with:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           ISSUE_NUMBER: ${{ github.event.issue.number }}
 
       - name: Reopen and comment on issue
-        if: ${{ steps.validateActor.outputs.isTeamMember && steps.checkDeployBlockers.outputs.HAS_DEPLOY_BLOCKERS == 'true' }}
+        if: ${{ steps.validateActor.outputs.isTeamMember == 'false' && steps.checkDeployBlockers.outputs.HAS_DEPLOY_BLOCKERS == 'true' }}
```

Comment:
This changed from `steps.validateActor.outputs.isTeamMember` to `steps.validateActor.outputs.isTeamMember == 'false'`- Was the logic wrong or should it be `true`?

Reply:
Good catch, that was a mistake it should have been `true`